src/ansys/mechanical/core/embedding/shims.py

```diff
@@ -1,40 +0,0 @@
-# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
-# SPDX-License-Identifier: MIT
-#
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in all
-# copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
-
-"""Shims for embedded Mechanical.
-
-These shims are used when APIs are released in newer versions of Mechanical,
-but workarounds exist in an older release
-"""
-
-
-def import_materials(
-    app: "ansys.mechanical.core.embedding.Application", material_file: str
-) -> None:
-    """Import material from matml file."""
-    if app._version >= 232:
-        materials = app.DataModel.Project.Model.Materials
```

Comment:
keep the shims - users might be using them.  just add an obsolete warning to it.

Reply:
This isn't correct, the jscript workaround is what we have to remove!

Reply:
I have updated and added tests as well.